cli/ballerina-packerina/src/main/java/org/ballerinalang/packerina/cmd/InitCommand.java

```diff
@@ -235,4 +241,27 @@ private String guessOrgName() {
         return guessOrgName;
     }
 
+    /**
+     * Validates the package name.
+     *
+     * @param pkgName The package name.
+     * @return True if valid package name, else false.
+     */
+    private boolean validatePkgName(PrintStream out, String pkgName) {
+        if (pkgName.isEmpty()) {
+           return true;
+        }
+        String validRegex = "^[a-zA-Z0-9_.]*$";
+        Pattern pattern = Pattern.compile(validRegex);
+        Matcher matcher = pattern.matcher(pkgName);
+        int count = 0;
```

Comment:
Cant we use pattern.matches() here instead of the counter ?

Reply:
Fixed with 4a203f0